src/app/themes/volumio/assets/variants/volumio/volumio-settings.json

```diff
@@ -15,5 +15,6 @@
   "knobThicknessDesktop": 0.2,
   "knobThicknessMobile": 0.09,
   "indexState": "playback",
-  "browseSourcesView": "grid"
+  "browseSourcesView": "grid",
+  "loadingBar": true
```

Comment:
I was unable to inject `uiSettingsService` into either the config, or the socket.service to enable or disable based on this flag. The app get stuck in an $apply loop. Any tips?